temp-legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/AmountModal.kt

```diff
@@ -495,13 +513,13 @@ fun KeypadCircleButton(
     text: String,
     testTag: String,
     textColor: Color = UI.colors.pureInverse,
-    fontSize: TextUnit = 24.sp,
-    btnSize: Dp = 64.dp,
+    fontSize: TextUnit = 32.sp,
+    btnSize: Dp = 92.dp,
```

Comment:
92.dp won't fit well on smaller devices. Can we make it based on the available space or leave it as it was?